docs/nada-ai-linear-model.md

```diff
@@ -101,7 +101,7 @@ def nada_main():
     result = my_model.forward(my_input)
 
     # Step 6: Produce the output for User with the variable name "my_output"
-    return result.output(user, "my_output")
+    return na.output(result, user, "my_output")
```

Comment:
there is a `from nada_dsl import` missing at the top

Reply:
:+1: